src/state/data/slice.ts

```diff
@@ -15,6 +15,8 @@ const CHAIN_SUBGRAPH_URL: Record<number, string> = {
   [SupportedChainId.OPTIMISM]: 'https://api.thegraph.com/subgraphs/name/ianlapham/optimism-post-regenesis',
 
   [SupportedChainId.POLYGON]: 'https://api.thegraph.com/subgraphs/name/ianlapham/uniswap-v3-polygon',
+
+  [SupportedChainId.CELO]: 'https://api.thegraph.com/subgraphs/name/joaquin-hernandez/uniswapv3-subgraph-celo',
```

Comment:
@ianlapham should this be transferred eventually?

Reply:
I think it should be transfered because Uniswap team will be responsible for maintenance over time, but if it happens relatively quickly after launch not a blocker (the longer we wait the longer itll take to sync new subgraph)